Formula/conan.rb

```diff
@@ -75,7 +129,7 @@ class Conan < Formula
 
   def install
     ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
-    %w[boto bottle cfgparse colorama fasteners monotonic passlib patch PyJWT PyYAML requests six].each do |r|
+    %w[backport_ipaddress boto bottle cffi cfgparse colorama cryptography enum34 fasteners idna ipaddress monotonic ndg-httpsclient passlib patch pyasn pyasn1 pycparser PyJWT pyOpenSSL PyYAML requests six].each do |r|
```

Comment:
If the order of these doesn't matter, let's simplify this by doing something like:

```ruby
resources.each do |r|
  r.stage do
    system "python", *Language::Python.setup_install_args(libexec/"vendor")
  end
end
```

If the order is important, let's at least make this a bit more readable by formatting the `%w[` style array like this:

```ruby
%w[
  backport_ipaddress
  boto
  bottle
  cffi
  (remaining resources)
].each do |r|
  # …
```